mathesar_ui/src/components/HyperlinkCard.svelte

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+  import { DropdownMenu } from '@mathesar-component-library';
+  import { iconMoreActions } from '@mathesar/icons';
+
+  export let href: string;
+  export let ariaLabel: string | undefined = undefined;
+
+  let isHoveringMenuTrigger = false;
+</script>
+
+<div class="hyperlink-card" class:hovering-menu-trigger={isHoveringMenuTrigger}>
```

Comment:
Since this is a common component, when used in other places might end up with invalid markup (button or links inside the wrapper link). This could happen majorly because its uses `slots` to render `top` and `bottom` contents and the consumer can pass a button or a link in those slots. 

Wouldn't it be better if we used something like this: https://svelte.dev/repl/40411730765d45c3a1b46c665562be70?version=3.55.1

Few minor things on the current implementation: 

1. I see a focus styling missing on the whole card but maybe this is something that we can handle when doing keyboard interactions. 
2. The hover styling of the vertical three dots menu doesn't seem to respect the border-radius of the card. This is something that you might want to handle in this PR itself.
<img width="361" alt="Screenshot 2023-01-12 at 10 04 40 AM" src="https://user-images.githubusercontent.com/11032856/211976993-ced7e508-0d69-462a-b3d3-78ab8aa0425d.png">

Reply:
> when used in other places might end up with invalid markup

- There are a number of things about your REPL code which confuse me. I think it might be best for us to hop on a call to hash this out. This table card has some nuances and tricky requirements. For example

    - (A) The whole card area needs to be clickable as a hyperlink, except the menu trigger.
    - (B) When hovering a truncated description, the user should see the popover from the truncation component.

    The combination of (A) and (B) lead me to the conclusion that the description needs to be a child of the hyperlink. I'm pretty sure that in your REPL we wouldn't be able to hover the table description to see its full value

    If this critique is something we could handle in a separate PR it might be nice, just so we can get this fix in before the release.

> focus styling missing on the whole card

- Yeah, I'd rather fix that separately from this PR. We have lots of other areas of the app that need similar improvements. I've opened #2248 to track that work.

> The hover styling of the vertical three dots menu doesn't seem to respect the border-radius of the card. 

- I've pushed ef85a568bbf40ea0f26c644201ce9d4713f5cb7b to fix that. It's weird to me that I wasn't seeing that hover problem when originally working on the PR (as is evident in my screenshots above). But I was indeed able to reproduce it now. Oh well! This fix ought to do the trick I think.